kubernetes-modes.el

```diff
@@ -42,6 +43,7 @@
     (define-key keymap [S-tab]     #'magit-section-cycle-global)
     ;; Misc
     (define-key keymap (kbd "q") #'quit-window)
+    (define-key keymap (kbd "Q") #'kubernetes-kill-buffers)
```

Comment:
Somehow this keybinding is not showing up for me after applying the patch,

Reply:
`kubernetes-mode-map` is set here via `defvar`, meaning that the value of `kubernetes-mode-map` will not change if already defined. You'll want to run `(makunbound 'kubernetes-mode-map)` first, then the new keybinding should appear fine.

Reply:
@jinnovation I am not sure why I need to do this after Emacs restart.

Reply:
@jinnovation I tried ```(makunbound 'kubernetes-mode-map)``` but still it is not working.

Reply:
I'm not quite sure why the binding isn't showing up after Emacs restart for you w/ this patch applied; the keybinding shows up fine for me.

Maybe I'll try writing a unit test that exercises the keybinding to see if that uncovers anything.

Reply:
I tried with ```emacs -Q```

Reply:
Do me a favor and try evaluating this `defvar` using `eval-defun`, as suggested in [this Stack Overflow post](https://emacs.stackexchange.com/a/2302/8244). I just did so and successfully got the `defvar` to re-evaluate to bind `Q` as `kubernetes-kill-buffers`.

Reply:
Evaluating using ```C-M-x``` did the magic ;). Not sure how we are going to make this work when Emacs loads.

Reply:
> Not sure how we are going to make this work when Emacs loads.

I can't seem to reproduce the issue you're describing, as the keybind shows up just fine for me on restarting Emacs w/ these changes applied.

<img width="436" alt="image" src="https://user-images.githubusercontent.com/1932579/120275955-0dea6680-c280-11eb-9dc2-9b8d8118fb6a.png">

<img width="428" alt="image" src="https://user-images.githubusercontent.com/1932579/120275938-088d1c00-c280-11eb-8cfb-ffa13761b389.png">

Reply:
ok

Reply:
Sounds good. Let me know if there's anything else you'd like expanded/changed w/ this PR.